chimney/src/main/scala/io/scalaland/chimney/Transformer.scala

```diff
@@ -43,6 +43,12 @@ trait Transformer[From, To] extends Transformer.AutoDerived[From, To] {
   */
 object Transformer extends TransformerCompanionPlatform {
 
+  /** Access an implicit `Transformer[From, To]`.
+    *
+    * @since 1.5.0
+    */
+  def apply[From, To](implicit t: Transformer[From, To]): Transformer[From, To] = t
```

Comment:
It's obvious that having a counterpart method for `PartialTransformer` would be beneficial. However, since there is already a defined method `PartialTransformer#apply`, adding the counterpart methods leads to a rather strange issue in Scala 2 (although it works well in Scala 3). See the minimized repro in Scastie — https://scastie.scala-lang.org/danicheg/eM7sBvzAS0q66azXmvayfg/6. Unfortunately, I haven't come up with a workaround yet, so I've left it as is.